clang/include/clang/Sema/Sema.h

```diff
@@ -4453,9 +4453,9 @@ class Sema final : public SemaBase {
                                       SourceLocation *ArgLocation = nullptr);
 
   /// Determine if type T is a valid subject for a nonnull and similar
-  /// attributes. By default, we look through references (the behavior used by
-  /// nonnull), but if the second parameter is true, then we treat a reference
-  /// type as valid.
+  /// attributes. We skip dependence then By default, we look through references
```

Comment:
```suggestion
  /// attributes. Dependent types are considered valid, so they can be checked during instantiation time. By default, we look through references
```

^^ make sure to clang-format.